rustfmt.toml

```diff
@@ -0,0 +1,8 @@
+# https://github.com/rust-lang/rustfmt/blob/master/Configurations.md
+format_code_in_doc_comments = true
+format_macro_matchers = true
+group_imports = "StdExternalCrate"
+imports_granularity = "Item"
```

Comment:
Personally I like the module granularity. Ideally there should be a Rust defined default everyone uses, but that seems far away.